lib/core/test/unit/Cardano/Wallet/DB/Arbitrary.hs

```diff
@@ -464,25 +467,25 @@ rootKeysSeq = unsafePerformIO $ generate (vectorOf 10 genRootKeysSeq)
     genRootKeysSeq :: Gen (ShelleyKey 'RootK XPrv)
     genRootKeysSeq = do
         (s, g, e) <- (,,)
-            <$> genPassphrase @"seed" (16, 32)
-            <*> genPassphrase @"generation" (0, 16)
+            <$> (SomeMnemonic <$> genMnemonic @12)
+            <*> (SomeMnemonic <$> genMnemonic @12)
             <*> genPassphrase @"encryption" (0, 16)
-        return $ Seq.generateKeyFromSeed (s, g) e
+        return $ Seq.generateKeyFromSeed (s, Just g) e
```

Comment:
Instead of forcing all second passphrases to be `Just`, it'd be better to `fmap SomeMnemonic` in the generator 2 lines above and actually generate a `Maybe SomeMnemonic`. This way, the generator would also cover cases where no second factor is provided, like before..

Reply:
> fmap SomeMnemonic

There is no arbitrary instance for `Mnemonic`, but I added a `frequency` combinator.

🤔 Could probably have added an `Arbitrary` instance… But I think this is fine too.